estate_classic/tests/test_estate_property_offer.py

```diff
@@ -0,0 +1,34 @@
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+
+
+@tagged('at_install')
+class EstateOfferTestCase(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.property = cls.env['estate_classic.property'].create({
+            "name": "Cool House",
+            "state": 'new',
+            "expected_price": 100000,
+        })
```

Comment:
The setup is the same between both test classes, you can inherit then
```suggestion
class EstateOfferTestCase(EstateTestCase):
```

Reply:
Is there no problem like duplicate function when we do that ? Because the parent's function will also be inherited no ?